src/components/Navigation.js

```diff
@@ -23,23 +22,43 @@ import Progress from "./Progress"
 const Navigation = observer(({ store }) => (
   <Layout>
     <InternalLink to={Home} store={store} >
-      <Icon path={mdiHome} color={store.currentPage == Home ? primary : black} size="1.8rem" />
+      <Icon
```

Comment:
Should this be a rule of thumb when you have 3 attributes?

Reply:
Yeah, I try to break up long lines as much as possible.

Any line over 80 characters is too long, according to many devs.

More practically, anything that you have to scroll sideways to see could be better presented on multiple lines.